tests/int/test_headers.py

```diff
@@ -0,0 +1,174 @@
+import pytest
+from pytest_check import check
+import requests
+import os
+import asyncio
+from api_test_utils.oauth_helper import OauthHelper
+from api_test_utils.apigee_api_apps import ApigeeApiDeveloperApps
+from api_test_utils.apigee_api_products import ApigeeApiProducts
+from time import time
+
+
+ENVIRONMENT = os.environ['ENVIRONMENT']
+SERVICE_BASE_PATH = os.environ['SERVICE_BASE_PATH']
+SERVICE_NAME = os.environ['FULLY_QUALIFIED_SERVICE_NAME']
+BASE_URL = f'https://{ENVIRONMENT}.api.service.nhs.uk/{SERVICE_BASE_PATH}'
+
+ID_TOKEN_TESTING_PRIVATE_KEY_PATH = os.environ["ID_TOKEN_TESTING_PRIVATE_KEY_ABSOLUTE_PATH"]
+
+HEADER_AUTHORIZATION = 'Authorization'
+HEADER_ECHO = 'echo'
+HEADER_BASE_URL = 'x-ers-network-baseurl'
+HEADER_USER_ID = 'x-ers-user-id'
+HEADER_X_REQUEST_ID = 'x-request-id'
+HEADER_ASID = 'xapi_asid'
+
+ORIGINAL_HEADERS = {
+    "CORRELATION_ID" : "x-correlation-id",
+    "BUSINESS_FUNCTION" : "nhsd-ers-business-function",
+    "ODS_CODE" : "nhsd-ers-ods-code",
+    "FILENAME" : "nhsd-ers-file-name",
+    "COMM_RULE_ORG" : "nhsd-ers-comm-rule-org",
+    "REFERRAL_ID" : "nhsd-ers-referral-id"
+}
+
+RENAMED_HEADERS = {
+    "CORRELATION_ID" : "nhsd-correlation-id",
+    "BUSINESS_FUNCTION" : "x-ers-business-function",
+    "ODS_CODE" : "x-ers-ods-code",
+    "FILENAME" : "x-ers-xapi-meta-file_name",
+    "COMM_RULE_ORG" : "x-ers-xapi-comm-rule-org",
+    "REFERRAL_ID" : "x-ers-xapi-meta-intended_ubrn"
+}
+
+@pytest.mark.smoke_test
+class TestHeaders:
+
+    @pytest.fixture
```

Comment:
I've considered throwing another fixture in combining the 2 just in case anyone wants separate plus it means that if the app fails to buiild then it won't affect the cleanup of the product

Reply:
If you were going to do that I'd suggest pulling it out to a conftest. Currently only using the combination. Here's an example of both which cleans up both https://github.com/NHSDigital/immunisation-history-api/blob/fe7beff968b34ecc4df2966c59e27efa0ba13593/tests/conftest.py#L263

Reply:
I've now moved the fixtures into conftest.py